src/app/containers/MostRead/index.jsx

```diff
@@ -22,9 +22,15 @@ const MostReadContainer = ({
 
   const { enabled } = useToggle('mostRead');
 
-  const mostReadEnabled = !isAmp && enabled && hasMostRead;
+  const mostReadToggleEnabled = enabled && hasMostRead;
 
-  if (!mostReadEnabled) {
+  // Do not render most read when a toggle is disabled
+  if (!mostReadToggleEnabled) {
+    return null;
+  }
+
+  // Do not render on AMP when there is no initial data
+  if (isAmp && !initialData) {
```

Comment:
We have the requirement that it's ok to server render this on the AMP most read _page_, but not when this is being used as a component on another page. Could we be explicit about that in these conditions? Maybe another prop we can pass in when we render this on the page to specify we want to server render this on AMP?

Reply:
Good shout Tom, I just remembered that we also plan on rendering SSR most read data for STY pages, so being explicit for what AMP page we want most read will prevent it from rendering on a AMP STY page. 👍 

I've added a `isMostReadPage` prop in my recent commit.

Reply:
Hey! Could we double check that server rendering most read on STY pages requirement? If that's the case, then we will want to render the same component that we are rendering on the page, and we run the risk of displaying stale data on the page. Otherwise, would we not want to use the amp-list implementation that you have been working on?